.changes/unreleased/Feature-20241007-111648.yaml

```diff
@@ -1,4 +1,3 @@
 kind: Feature
-body: add optional "use_docker" input to send deploy via docker, uses OpsLevel CLI
-  otherwise
+body: support sending deploy using Docker or OpsLevel CLI directly via "use_docker" input arg, "use_docker" defaults to true
```

Comment:
Breaking change?  Also should we put the bigger explanation in here or are we going to do that in the final release notes?